docs/api/index.md

```diff
@@ -6,45 +6,16 @@ nav_order: 3
 ---
 
 # Election Information API
+For getting region-specific information about upcoming elections
 
-The Election Information API powers the VoteAmerica website and suite of tools.
+# Action API
+For learning about the actions a voter has taken
 
-This API is available as part of the VoteAmerica premium tools. You can [learn more about our premium tools here](https://www.readysetvote.io/).
+# Registration API
+For registering voters
 
-This data has been gathered and vetted by our research staff, but if you see an error please [report it](https://www.voteamerica.com/report-incorrect-info/).
+# Locate API
```

Comment:
Since we don't have docs for this and [don't offer it externally](https://voteamerica.slack.com/archives/C037XNRME13/p1648233348166669?thread_ts=1648231990.019279&cid=C037XNRME13) anyway, thinking we should remove Locate and Verify.

Reply:
Agreed

Reply:
Or maybe add "docs coming soon"? I like acknowledging that the APIs exist

Reply:
> docs coming soon

Are they?

My inclination is still to leave them out, but don't feel super strongly.

Reply:
They have to be, we won't be able to do the partnerships we want with without making the docs, and we may as well host them